qutebrowser/mainwindow/statusbar/command.py

```diff
@@ -216,3 +218,15 @@ def sizeHint(self):
             text = 'x'
         width = self.fontMetrics().width(text)
         return QSize(width, height)
+
+    @pyqtSlot(str)
+    def _incremental_search(self, text):
+        if config.val.search.incremental:
+            search_prefixes = {
+                '/': 'search -- ',
+                '?': 'search -r -- ',
+            }
+
+            # `if text` check is for when user presses <Esc> and `len(text)` is 0
+            if self.prefix() in '/?' and text:
```

Comment:
Hmm, `'' in '/?'` being `True` is a quite subtle difference between that and `'' in ['/', '?']`, I never realized that!

I think it'd be nicer to rewrite this to `if self.prefix() in ['/', '?']:` instead, then with an empty `text`, `self.prefix()` will return `''` and that will be `False`.